VRCSubs/vrcsubs.py

```diff
@@ -183,12 +183,19 @@ def _process_osc(self):
 def main():
     global config
     # Load config
-    cfgfile = f"{os.path.dirname(os.path.realpath(__file__))}/Config.yml"
-    if os.path.exists(cfgfile):
+    cfgfile = f"{os.getcwd()}/config.yml"
+    # Check if config file exists, create a file if not
+    if os.path.exists('config.yml'):
```

Comment:
Please make this `if os.path.exists(cfgfile):`  to account for the above variable being made